Demo-Mac/ShoppingItem.m

```diff
@@ -73,5 +73,5 @@ - (void) setPicture:(NSImage *)picture {
         }
     }
     NSCAssert(bitmapRep != nil, @"No bitmap rep");
-    return [bitmapRep representationUsingType: NSJPEGFileType properties: nil];
+    return [bitmapRep representationUsingType: NSJPEGFileType properties: @{}];
```

Comment:
This change is due to the fact that `representationUsingType:properties:` is annotated to not accept `nil` for the properties dictionary.